docs/contributing/index.md

```diff
@@ -77,7 +82,7 @@ After the PR moves from **Draft** to **Ready for review**, the CI status checks
 
 To view a deployed preview on a **Ready for review** PR, click the **Details** link on the netlify commit status line:
 
-![deploy-preview](./deploy-preview.png)
+<!-- ![deploy-preview](./deploy-preview.png) -->
```

Comment:
oh, I see, are we commenting this image out because it is causing Netlify to fail? 

Because the preview doesn't work? teach me!

Reply:
There is no `deploy-preview.png`, so the website can't be built. We either have to comment this line out until we add the image in question or delete the line altogether.

Reply:
I hate to admit it, but I learn best by making mistakes! You fixed another one of my errors, and I am learning. Thank you. I restored the missing image, removed the comment. I see what you did for the front matter and rename. thank you thank you.